### ISSUE TITLE
Enable reacting to censored message to delete message

### ISSUE THREAD

**Issue by cbrxyz** [cbrxyz/pi-bot#137]
Allow the original poster of the censored message to react to the censored message with a certain emoji to delete the emoji.

**Comment by Nydauron** [comment]
I'd assume we'd use `:wastebasket:` (🗑️) but as for questions of implementation, do you think it is better to have the bot react to the censored message to allow for easier access? The only con I can see is that other users could confuse this as a general vote to remove the message.

**Comment by cbrxyz** [comment]
No, because that would result in a lot of trash can emojis that just don't need to be there. I can see what you're saying, but I'm not sure about that because of space reasons.
